pyproject.toml

```diff
@@ -52,6 +52,7 @@ termcolor = "^1.1"
 packaging = ">=19,<21"
 tomlkit = "^0.5.3"
 jinja2 = "^2.10.3"
+setuptools = ">=47.3.1"
```

Comment:
What would be a more appropriate version constraint? (this is latest on pypi as of today)